packages/main/src/plugin/container-registry.ts

```diff
@@ -1692,7 +1692,11 @@ export class ContainerProviderRegistry {
     }
   }
 
-  async createAndStartContainer(engineId: string, options: ContainerCreateOptions): Promise<{ id: string }> {
+  async createAndStartContainer(
+    engineId: string,
+    options: ContainerCreateOptions,
+    doStartContainer: boolean,
+  ): Promise<{ id: string }> {
```

Comment:
Probably need to rename the method createContainer() (as it is not starting all the time)

(we can still have aliases createAndStart that does the start)